Admin/AbstractAdmin.php

```diff
@@ -227,6 +227,10 @@
     /**
      * The base code route refer to the prefix used to generate the route name.
      *
+     * NEXT_MAJOR: remove this attribute.
+     *
+     * @deprecated This attribute is deprecated since 3.31 and will be removed in 4.0
```

Comment:
Please prefer 3.x over 3.31, you don't know for sure that this will be released in 3.31, and people probably don't need that level of detail anyway ;)